src/components/ItemPage/Shamazin.jsx

```diff
@@ -24,14 +26,21 @@ class Shamazin extends React.Component{
     componentDidMount(){
         ShamazinService.getItemById(this.state.itemID)
             .then(response=>{
-                this.setState({item:response.data})
-            })
+                this.setState({item:response.data});
+            });
+            
+            ShamazinService.getFamilyId(this.state.itemID)
+            .then( response=>{
+                console.log("FamilyId(Shamazin): "+ response.data);
```

Comment:
Will we need console logs for final product? Or were you just using this for manual testing purposes

Reply:
Dang I tried to get rid of all of them, just missed this one I'll make sure to remove it